cypress/tests/zTemp.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+const helpers = require('../support/Helpers')
+const homePage = require('../support/components/HomePage')
+const modelPage = require('../support/components/ModelPage')
+const train = require('../support/Train')
+const trainDialogsGrid = require('../support/components/TrainDialogsGrid')
+const editDialogModal = require('../support/components/EditDialogModal')
+
+Cypress.TestCase('zTemp', 'Temporary Experimental Test', zTemp)
+function zTemp() {
+  cy.task('exists', 'C:/repo/ConversationLearner-UI/cypress/integration/Tools').then(fileExists => {console.log(`fileExists: ${fileExists}`)})
```

Comment:
Shouldn't use hard-coded paths. Only works on your machine

Reply:
This is a temporary test case, it is used to experiment, it was not intended for others to run it. Normally I would not use a hard-coded path like this. However I did just fix it.